quickstartKotlin/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,5 @@
     <string name="max_audio_bitrate">Max Audio Bitrate (bits per second)</string>
     <string name="max_video_bitrate">Max Video Bitrate (bits per second)</string>
     <string name="vp8_simulcast">VP8 Simulcast</string>
+    <string name="turn_speaker_off">Speaker OFF</string>
```

Comment:
Not a big deal, but I'd say move this under turn_speaker_on for better organization. Up to you though.